services/src/main/java/org/jboss/windup/web/services/rest/ApplicationGroupEndpointImpl.java

```diff
@@ -47,13 +47,9 @@
     @Override
     public Collection<ApplicationGroup> getApplicationGroups(Long projectID)
     {
-        MigrationProject project = entityManager.find(MigrationProject.class, projectID);
-
+        final MigrationProject project = entityManager.find(MigrationProject.class, projectID);
         if (project == null)
-        {
-            throw new NotFoundException("MigrationProject not found");
-        }
-
+            throw new IllegalArgumentException("MigrationProject not found, ID:  " + projectID);
```

Comment:
Well now I figured out: Originally there was no handling for this case, so I added that. Then someone added it, and my change rolled with rebases. I guess OK to remove